src/components/utils/CalciteScrim.tsx

```diff
@@ -16,10 +16,22 @@ const divStyle = {
 
 interface ScrimProps {
   loading: boolean;
+  disabled: boolean;
 }
 
-export const CalciteScrim: FunctionalComponent<ScrimProps> = ({ loading }) => (
-  <div style={divStyle}>{loading ? <calcite-loader is-active></calcite-loader> : null}</div>
-);
+const containerDivStyle = {
+  position: "relative",
+  zIndex: "1"
+};
+
+export const CalciteScrim: FunctionalComponent<ScrimProps> = ({ loading, disabled }, children) =>
+  disabled || loading
```

Comment:
moved this into the functional component because it was repetitive throughout the other components. It won't return anything if its not disabled or loading.